Client/SafeBrowsingPages/Phishing.html

```diff
@@ -0,0 +1,216 @@
+            display: flex;
+            flex-wrap: wrap;
+            flex-direction: column;
+            align-content: flex-start;
+        }
+
+        .hidden {
+            display: none;
+        }
+
+        i {
+            border: solid white;
+            border-width: 0 2px 2px 0;
+            display: inline-block;
+            padding: 5px;
+            margin-left: 7px;
+        }
+
+        .up {
+            transform: rotate(-135deg);
+            transition: 0.3s;
+            -webkit-transform: rotate(-135deg);
+            margin-bottom: -4px;
+        }
+
+        .down {
+            transform: rotate(45deg);
+            transition: 0.3s;
+            -webkit-transform: rotate(45deg);
+            margin-bottom: 2px;
+        }
+
+        /** Center the content for iPads **/
+        @media (min-width: 600px) and (min-height: 600px) {
+            .content {
+                margin: 0;
+                position: absolute;
+                top: 50%;
+                left: 50%;
+                max-width: 650px;
+                -ms-transform: translate(-50%, -50%);
+                transform: translate(-50%, -50%);
+            }
+        }
+    </style>
+    <script>
+        function toggleDetails() {
+            var element = document.getElementById("details");
+            var arrow = document.getElementById("arrow");
+            if (element.classList.contains("hidden")) {
+                element.classList.remove("hidden");
+                arrow.classList.replace("down", "up");
+            } else {
+                element.classList.add("hidden");
+                arrow.classList.replace("up", "down");
+            }
+        }
+
+        function backToSafety() {
+            webkit.messageHandlers.InterstitialPageHandler.postMessage("go_back");
+        }
+
+        function proceedUnsafe() {
+            webkit.messageHandlers.InterstitialPageHandler.postMessage("proceed_unsafe");
+        }
+    </script>
+</head>
+
+<body class="background content">
+    <div class="container post">
+        <svg class="icon" xmlns="http://www.w3.org/2000/svg">
+            <g fill="rgb(250, 251, 251)">
+                <path
+                    d="M58.584 59.167H5.414a2.423 2.423 0 01-2.116-3.589L29.86 7.158c.848-1.544 3.385-1.544 4.232 0l26.355 48.045a2.42 2.42 0 01-1.862 3.964zM31.975 13.35L9.498 54.325h44.954L31.975 13.35zm0 11.922a2.418 2.418 0 012.416 2.421V39.8a2.42 2.42 0 01-2.416 2.421 2.417 2.417 0 01-2.413-2.421V27.694a2.416 2.416 0 012.413-2.42zm1.721 20.069c.434.46.7 1.09.7 1.72 0 .63-.266 1.258-.724 1.718a2.437 2.437 0 01-1.69.703 2.486 2.486 0 01-1.714-.703 2.433 2.433 0 01-.702-1.718c0-.63.242-1.26.702-1.72.891-.896 2.535-.896 3.428 0z" />
+            </g>
+        </svg>
+
+        <p class="title">
+            Deceptive site ahead
+        </p>
+
+        <p class="description">
+            Attackers on testsafebrowsing.appspot.com may trick you into doing something dangerous like installing
```

Comment:
`testsafebrowsing.appspot.com` here should be the hostname of the website, not a hard-coded test string :)

Also "detected phishing" should be a link to https://transparencyreport.google.com/safe-browsing/search?url=https%3A%2F%2Ftestsafebrowsing.appspot.com%2Fs%2Fphishing.html (replacing and escaping the real URL).